internal/cloud/metadata_test.go

```diff
@@ -37,6 +37,16 @@ func dmidecodeGCP() []byte {
 	return []byte("Google")
 }
 
+func dmidecodeNutanix() []byte {
+	return []byte(`
+		SomeUselessProp: some-value-1.1.0
+
```

Comment:
Delete this empty line maybe?

Reply:
It was intended to make it clear that dmidecode output will very likely have some extra content not related to nutanix.

Yet, i don't mind removing it.